### ISSUE TITLE
New tag?

### ISSUE THREAD

**Issue by dkarrasch** [JuliaGeometry/GeometricalPredicates.jl#28]
It seems that `GeometricalPredicates.jl` requires a new tag to work under Julia v1.0.

**Comment by skariel** [comment]
yep, I'll look into this

**Comment by dkarrasch** [comment]
I just checked: the master branch seems to be loadable in Julia 1.0, and tests pass once you change `using Base.Test` to simply `using Test`.

**Comment by dkarrasch** [comment]
Things are getting serious, [see here](https://discourse.julialang.org/t/package-compatibility-caps/15301). I can't load neither `GeometricalPredicates.jl` nor `VoronoiDelaunay.jl` because of this. Can I help with anything, @skariel ?

**Comment by skariel** [comment]
@dkarrasch yeah sure, if you want to tag etc. go ahead, I'm adding you as collaborator :)

**Comment by skariel** [comment]
@dkarrasch also adding you to VoronoiDelaunay

**Comment by skariel** [comment]
@dkarrasch thanks!
